### PR TITLE
Fix labels and job count for projects with zero operations.

### LINKED ISSUE #454: Flow status returns no jobs when no operations
### Description

For a project with initialized jobs but no defined operations, `flow.print_status()` returns that there are no jobs.

### To reproduce

Create an empty flow project and initialize jobs, then run `print_status()`.

### Error output

```

Overview:

Total # of jobs: 0

label
-------


operation
-----------

```

### System configuration

Please complete the following information:

 - Version of signac-flow: 0.12

### PR DESCRIPTION
**Pull request by bdice**
## Description
The new status fetching code did not correctly handle FlowProjects with zero operations. This resolves #454.

I don't really want to write tests for status checks in this PR but it should be done at some point.

I tested this manually by creating a project with no operations and with/without labels. I tested `python project.py status` with and without a bunch of flags, and it seems to be fine now.

## Types of Changes
- [ ] Documentation update
- [x] Bug fix
- [ ] New feature
- [ ] Breaking change<sup>1</sup>

<sup>1</sup>The change breaks (or has the potential to break) existing functionality.

## Checklist:
- [x] I am familiar with the [**Contributing Guidelines**](https://github.com/glotzerlab/signac-flow/blob/master/CONTRIBUTING.md).
- [x] I agree with the terms of the [**Contributor Agreement**](https://github.com/glotzerlab/signac-flow/blob/master/ContributorAgreement.md).
- [x] My name is on the [list of contributors](https://github.com/glotzerlab/signac-flow/blob/master/contributors.yaml).
- [x] My code follows the [code style guideline](https://github.com/glotzerlab/signac-flow/blob/master/CONTRIBUTING.md#code-style) of this project.
- [ ] The changes introduced by this pull request are covered by existing or newly introduced tests.

If necessary:
- [ ] I have updated the API documentation as part of the package doc-strings.
- [ ] I have created a separate pull request to update the [framework documentation](https://docs.signac.io/) on [signac-docs](https://github.com/glotzerlab/signac-docs) and linked it here.
- [x] I have updated the [changelog](https://github.com/glotzerlab/signac-flow/blob/master/changelog.txt).

### CODE REVIEW

**Review by kidrahahjo — APPROVED** [review]
This is an interesting find. I think we might need to do regression tests for status checks. I did not write the status tests for aggregation in #464 because we'll have to rewrite most of the status checks if we go for regression testing. For now, we can merge this and handle tests as part of #435 in a separate PR.

File: flow/project.py [hunk 1]
```diff
@@ -2336,20 +2336,24 @@ def compute_status(data):
...
+            # unique jobs from the aggregates containing only one job.
+            if len(single_operation_groups) > 0:
+                individual_jobs = {
+                    aggregate_group[3][0]
+                    for aggregate_group in aggregate_groups
+                    if len(aggregate_group[3]) == 1
+                }
+            else:
+                # If no operations exist, use all jobs in the project.
+                individual_jobs = set(self)
```
Comment (vyasr) [inline flow/project.py:2356]: Why did the location of this block move? This change doesn't seem necessary.

**Review by vyasr — CHANGES_REQUESTED** [review]
This PR looks fine to me (just one question, but it's not holding anything back). I didn't approve yet, because I would like us to have a record somewhere of what tests are known to be missing and maybe start making a plan for them. In the push to get the last release ready we made a lot of changes, and while we actually improved coverage in some cases, we almost certainly degraded it in others. Can you open an issue we can use as a checklist for these kinds of things that we find as aggregation gets merged and we continue refactoring? And @kidrahahjo can you populate it with other cases that you're aware need to be tested? That would be very helpful, and I'll approve this PR once that's done.

**Comment by kidrahahjo** [comment]
All the options like `--detailed`, `--stack`, `-1`, `--pretty`, `--full`, `--no-overview`, `--parameters`, etc. must be tested (regression tests like submission scripts).
Here is a list of tests: 
- [ ] Test status output when only labels are present
- [ ] Test status output when using **only** aggregation in flow project.
- [ ] Test status output when using aggregation with non-aggregate operations in flow project.
- [ ] Test status with parameters when aggregation is used.
- [ ] Once labels for aggregation is implemented, test labels for aggregates.

File: flow/project.py (hunk 1 above)
Comment (vyasr) [reply flow/project.py:2355]: @bdice any comments?

**Comment by vyasr** [comment]
@kidrahahjo can you move that checklist to a separate issue? Then we can merge this PR.

File: flow/project.py (hunk 1 above)
Comment (bdice) [reply flow/project.py:2355]: I just wanted to move the construction of `individual_jobs` closer to the place where it's actually used.

**Review by vyasr — APPROVED** [review]

**Pull request merged**